docs/awscodedeploy

```diff
@@ -2,13 +2,29 @@ This service lets you deploy apps in [AWS CodeDeploy][1] when Deployments are cr
 
 You need to provide an AWS access key id and the corresponding secret access key having at least the permission for the `codedeploy:CreateDeployment` action. This is the minimal required policy file:
 
+Note: Replace '123ACCOUNTID' with your account id. As well as entering the correct Application name, deployment groups, and region.
```

Comment:
Can you change this to:
`You will need to replace “us-east-1” if you are using a different region, and replace “123ACCOUNTID” with your AWS account ID that is found on your Account Settings page.` from the AWS docs? Thanks!